index.php

```diff
@@ -0,0 +1,60 @@
+<?php
+//Connect to Database:
+
+
+$db = new PDO('mysql:host=db; dbname=skateplazas', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
You should move this code into a nice reusable function called `getDbConnection` or something like that, you can then use this function on any page that requires a database connection, for example an edit/add page.